CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ All Sniffnet releases with the relative changes are documented in this file.
 
 ## [UNRELEASED]
 
+- Added app `id` in order to correctly show the icon and app name on Linux Wayland ([#292](https://github.com/GyulyVGC/sniffnet/issues/119))   
```

Comment:
issue number and link should be consistent :P

Reply:
Yes, fixed.